package.json

```diff
@@ -14,8 +14,11 @@
     "dotenv": "^8.0.0",
     "express": "^4.17.0",
     "express-handlebars": "^3.1.0",
+    "mysql": "^2.17.1",
     "mysql2": "^1.6.5",
-    "sequelize": "^5.8.6"
+    "nodemon": "^1.19.2",
```

Comment:
nodemon should be a dev dependency

Reply:
Could not figure out the watch script. Will address in next PR